hack/verify-godeps.sh

```diff
@@ -92,7 +92,9 @@ fi
 # Godeps/_workstapces/src/github.com/fsouza/go-dockerclient/testing/data/symlink'
 # is an intentionally broken symlink. Linux can use --no-dereference. OS X cannot.
 # So we --exclude='symlink' so diff -r doesn't die following a bad symlink.
-if ! _out="$(diff -Naupr --exclude='symlink' ${KUBE_ROOT}/Godeps/_workspace/src ${_kubetmp}/Godeps/_workspace/src)"; then
+# godep doesn't include test files by default or when using `godep update` so
```

Comment:
`-t` only works with `godep save` so if you do a `godep update`, as I did for the cadvisor bump, test files are deleted. Why would you want to keep test files & test data vendored anyway?